src/web-integration-step-by-step.md

```diff
@@ -0,0 +1,108 @@
+
+### NHS Login - Authorize call
+The NHS requires suppliers to pass prompt=none to the NHS Login `/authorize` endpoint. This will ensure that user is shown an error page if there is an SSO issue.
+
+### NHS Login - Handle user not sharing  data
+NHS login require suppliers to handle the return from this page where users are asked to agree to sharing their login details with the supplier. NHS Login will return to the suppliers auth-return endpoint with `error=access_denied` and `error_description=ConsentNotGiven`.
+
+If the user selects "I do not want to use my NHS login" the supplier should present a suitable message, ideally with a link to the NHS App home page.
+
+## Site Links <a name="sitelinks"></a>
+
+It iss possible to allow users to open links that suppliers send out (e.g. via email or sms) within the NHS App. This requires changes on both the NHS App and the supplier's application. Suppliers should work with the NHS App onboarding team to co-ordinate the changes.
+
+### Android
+
+Site links are achieved on Android using [Android App Links](https://developer.android.com/training/app-links/verify-site-associations). This will require the supplier to host a *Asset Links* file on `https://<domain-name>/.well-known/assetlinks.json` with the contents:
+
+```json
+[
+    {
+        "relation": [
+            "delegate_permission/common.handle_all_urls"
+        ],
+        "target": {
+            "namespace": "android_app",
+            "package_name": <package-name>,
+            "sha256_cert_fingerprints": [
+                <cert-fingerprint>
+            ]
+        }
+    }
+]
+```
+
+where `package-name` and `cert-fingerprint` vary per NHS App environment. These values will be provided by the NHS App team during integration.
+
+### iOS
+
+Site links are achieved on iOS using [Associated Domains](https://developer.apple.com/documentation/safariservices/supporting_associated_domains). This will require the supplier to host a *Site Association* file on `https://<domain-name>/.well-known/apple-app-site-association` with the contents:
+
+```json
+{
+  "applinks": {
+    "apps": [],
+    "details": [
+      {
+        "appID": "<app-id>",
+        "paths": [ <paths>]
+      }
+    ]
+  }
+}
+```
+
+where app-id will vary per NHS App environment and will be provided by the NHS App team during integration. Paths will contain a list of url paths that the supplier would like to open on the NHS App.
+
+## Limitations <a name="limitations"></a>
+
+Although, the NHS App iOS and Android apps act largely as web browsers - there are some web functions that are not available. In some cases these will be addressed by the [NHS App JS API](/nhsapp-developer-documentation/js-api-specification).
+
+| Action             | Notes/Workarounds |
+|--------------------|-------------------|
+| File Download      | Currently file down is not possible in web integrations. It is currently planned that a method will be added to the NHS App JS Api to download a file.|
```

Comment:
just checking.. is this definitely a thing planed for the future?  if so, should it be added to the specification with a status of planned

Reply:
Yeah its planned but still awaiting some IG signoff - have updated the comment to show its in consideration